aep/general/0148/aep.md.j2

```diff
@@ -1,5 +1,155 @@
 # Standard fields
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/28) for more
-information.
+Certain concepts are common throughout any corpus of APIs. In these situations,
+it is useful to have a standard field name and behavior that is used
+consistently to communicate that concept.
+
+## Guidance
+
+Standard fields **should** be used to describe their corresponding concept, and
+**should not** be used for any other purpose.
+
+### Resource paths and IDs
+
+#### path
+
+Every resource **must** have a `string path` field, used for the resource path
+(AEP-122), which **should** be the first field in the resource.
+
+#### parent
+
+The `string parent` field refers to the resource name of the parent of a
+collection, and **should** be used in most `List` (AEP-132) and `Create`
+(AEP-133) requests.
+
+### Names
+
+#### display_name
+
+The `string display_name` field **must** be a mutable, user-settable field
+where the user can provide a human-readable name to be used in user interfaces.
+[Declarative-friendly resources][] **should** include this field.
+
+Display names **should not** have uniqueness requirements, and **should** be
+limited to <= 63 characters.
+
+**Note:** The `_name` suffix **should not** be used to describe other types of
+names unless otherwise covered in this AEP.
+
+#### title
```

Comment:
FWIW I kind of feel like this is a weird standard field to have. we have display name already. What does it mean to be "more formal"?

I'd vote to remove, but if we can't agree we can just file an issue.

Reply:
Yeah I don't know about this one.  Maybe we can tighten up the wording rather than remove it?  I assume it was written for a reason.

FWIW, `User.title` isn't a good substitute for `User.display_name`, but `Article.display_name` and `Job.display_name` both seem like poor substitutes for `.title`.  It seems like a real distinction to me, even if the AEP is hand-wavy about what that distinction is.

Filed https://github.com/aep-dev/aep.dev/issues/153.